### ISSUE TITLE
const and let is still allowed

### ISSUE THREAD

**Issue by finalclass** [nkt/eslint-plugin-es5#8]
I'm using your es5 plugin which is very nice however eslint still is not complaining about `const` and `let`.
Unfortunately our environment does not support `const` nor `let` (which come from es6)

**Comment by finalclass** [comment]
@macklinu thank you!
With this rule it works perfectly fine.
This is my test config:

*.eslintrc.json*
```json
{
    "plugins": [ "es5" ],
    "extends": [ "plugin:es5/no-es2015" ],
    "parserOptions": { "sourceType": "module" },
    "rules": { "es5/no-block-scoping": "error" }
}
```

and a test file:

*index.js*
```js
const a = 5;
let b = 10;
